bin/reth/src/lib.rs

```diff
@@ -42,4 +43,13 @@ struct NetworkOpts {
     /// Will fall back to a network-specific default if not specified.
     #[arg(long, value_delimiter = ',')]
     bootnodes: Option<Vec<NodeRecord>>,
+
+    /// The path to the known peers file. Connected peers are
+    /// dumped to this file on node shutdown, and read on startup.
+    #[arg(long, value_name = "FILE", verbatim_doc_comment, default_value_t)]
+    peers_file: PlatformPath<KnownPeersPath>,
+
+    /// Do not persist peers. This ignores the --peers-file option.
+    #[arg(long, verbatim_doc_comment)]
```

Comment:
should we use `conflicts_with = "peers_file"`?

Reply:
That's a good idea! If both options are used, it's probably a mistake and should be notified to the user.